app/middleware/cache_headers.py

```diff
@@ -29,7 +29,10 @@ def __call__(self, request):
         if request.path in ['/checker', '/get-token']:
             # never cache the /checker and /get-token endpoints.
             add_never_cache_headers(response)
-        elif request.method in ('GET', 'HEAD'):
+        elif (
+            request.method in ('GET', 'HEAD') and
+            not request.path.startswith(f'/api/stac/{settings.STAC_VERSION}/static/')
```

Comment:
you might rather want to use `STATIC_URL` from settings.py

Reply:
...and what just pops into my mind: shouldn't we rather use `/api/stac/static` and omit the version info here? If we ever have to support v1.0.0 it will only affect the REST part, not the admin / static etc. (/admin is already accessible under /api/stac/static). Only issue is with spec, for which I don't have a good solution right now... (maybe /api/stac/static/v0.9.0/api.html).

Reply:
That's a good point, but I suggest to create a new jira ticket to address this. I'll use the `STATIC_URL` as mentioned above for this PR though. But leave the change of moving the `STAC_VERSION` in settings if you don't mind @boecklic ?

Reply:
Ticket for the static path: https://jira.swisstopo.ch/browse/BGDIINF_SB-1610